### PR TITLE
chore: add makefile and documentation

### PR DESCRIPTION
**Pull request by raphapr**
* add descriptions to terraform input variables
* add makefile with remote commands in order to ease build machine tasks
* add some documentation as well

### CODE REVIEW

File: .rsyncignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dev_cycle
```
Comment (stijnopheide) [inline .rsyncignore:1]: not sure that this file needs to be in source control.

Comment (stijnopheide) [reply .rsyncignore:1]: oh, I see, it does. sorry :)

File: README.md [hunk 2]
```diff
@@ -21,13 +21,95 @@ This repository provides a terraform module that sets up the required infrastruc
...
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
```
Comment (stijnopheide) [inline README.md:106]: I know this is from before you started on the project, but I can't remember why it was necessary to create this role outside of the dev_cycle module. Should we move it in the module and if it is required to give specific access export it as output?

Comment (raphapr) [reply README.md:106]: Oh I see. Yes, good point. I'll definitely do that but I think it would be nice put this change in a new PR.

Comment (raphapr) [reply README.md:106]: https://github.com/vouch-opensource/linuxkit-image-development/pull/16

**Review by stijnopheide — APPROVED** [review]

**Comment by raphapr** [comment]
I did rebase and update README with the new input variable `vmimport_service_role_enabled` and new example

**Pull request merged**